Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/transform/AuditTransform.java

```diff
@@ -0,0 +1,579 @@
+     * @return
+     */
+    protected EventIdentificationType createEventIdentification(String serviceName, boolean isRequesting) {
+        CodedValueType eventID = createCodeValueType(getEventIdCode(serviceName), null,
+            getEventCodeSystemName(serviceName), isRequesting ? getEventDisplayNameRequestor(serviceName) : getEventDisplayNameResponder(serviceName));
+
+        EventIdentificationType oEventIdentificationType = getEventIdentificationType(eventID, isRequesting, serviceName);
+        oEventIdentificationType.getEventTypeCode().add(AuditDataTransformHelper.createCodeValueType(getEventTypeCode(serviceName), null,
+            getEventTypeCodeSystem(serviceName), getEventTypeCodeDisplayName(serviceName)));
+        return oEventIdentificationType;
+    }
+
+    /**
+     * Create the ActiveParticipant for an audit log record.
+     *
+     * @param userInfo
+     * @param userIsReq
+     * @return
+     */
+    protected AuditMessageType.ActiveParticipant createActiveParticipantFromUser(UserType userInfo,
+        Boolean userIsReq) {
+        AuditMessageType.ActiveParticipant participant = new AuditMessageType.ActiveParticipant();
+
+        // Set the User Id
+        String userId = null;
+        if (userInfo != null && userInfo.getUserName() != null && userInfo.getUserName().length() > 0) {
+            userId = userInfo.getUserName();
+            participant.setUserID(userId);
+        }
+
+        // If specified, set the User Name
+        String userName = null;
+        if (userInfo != null && userInfo.getPersonName() != null) {
+            if (userInfo.getPersonName().getGivenName() != null && userInfo.getPersonName().getGivenName().length() > 0) {
+                userName = userInfo.getPersonName().getGivenName();
+            }
+
+            if (userInfo.getPersonName().getFamilyName() != null
+                && userInfo.getPersonName().getFamilyName().length() > 0) {
+                if (userName != null) {
+                    userName += (" " + userInfo.getPersonName().getFamilyName());
+                } else {
+                    userName = userInfo.getPersonName().getFamilyName();
+                }
+            }
+            if (userName != null) {
+                participant.setUserName(userName);
+            }
+        }
+
+        participant.setUserIsRequestor(userIsReq);
+        return participant;
+    }
+
+    /**
+     *
+     * @param eventID
+     * @param isRequesting
+     * @param serviceName
+     * @return
+     */
+    protected EventIdentificationType getEventIdentificationType(CodedValueType eventID, boolean isRequesting, String serviceName) {
+        EventIdentificationType oEventIdentificationType = createEventIdentification(
+            isRequesting ? getEventActionCodeRequestor(serviceName) : getEventActionCodeResponder(serviceName),
+            AuditTransformConstants.EVENT_OUTCOME_INDICATOR_SUCCESS, eventID);
+        return oEventIdentificationType;
```

Comment:
Don't create a placeholder object just to immediately return it; change this to an inline return.